templates/open_pr.html

```diff
@@ -2,9 +2,58 @@
 
 {% block content %}
 
+
 <div class="container-fluid">
-    <h1>Open PR</h1>
-    <p>bla bla bla 123</p>
+    <h1 class="mt-4">Open Pull Requests <button id="update_button" type="button" class="btn btn-outline-primary"
+            style="display: inline-flex; align-items: center; width: 120px; margin-bottom: 20px;">
+            Update data
+        </button></h1>
+
+
+    <table id="example" class="display" style="width:100%">
+        <thead>
+            <tr>
+                <th>Repository</th>
+                <th>PR</th>
+                <th>Draft / WIP</th>
+                <th>Author</th>
+                <th>Created at</th>
+            </tr>
+        </thead>
+        <tbody>
+            {% for repo, prs in github_open_pr.items() %}
+            {% for pr in prs %}
+            <tr>
+                <td>{{ repo }}</td>
+                <td>
+                    <a href="{{ pr.html_url }}" target="_blank">PR#{{ pr.number }}</a> {{ pr.title }}
+                </td>
+                <td>
+                    {% if pr.draft or "WIP" in pr.title %}
+                        yes
+                    {% endif %}
+                </td>
+                <td>{{ pr.user_login }}</td>
+                <td>{{ pr.created_at|format_time }}</td>
+            </tr>
+            {% endfor %}
+            {% endfor %}
+        </tbody>
+
+    </table>
 </div>
 
+<!-- JavaScript -->
+{% block js %}
+<script src="../static/js/open_pr.js"></script>
```

Comment:
`url_for` was better than hardcoded address and what I was proposing was using [{% include %}](https://jinja.palletsprojects.com/en/3.0.x/templates/#include).

Reply:
I think the `include` tag is used only for html templates https://jinja.palletsprojects.com/en/3.0.x/templates/?highlight=include#include

Reply:
I believe you can use Jinja2 for whatever text files you want. Using include here is not a hard requirement so feel free to just reverse the change back to the url_for and forget about the idea.

Reply:
done